content/quiz_events.md

```diff
@@ -95,6 +95,40 @@ Below are the the scoring calculation rules and the logic for them. This procedu
 **Team Roster**
 : This is an array of quizzer objects, each of which contains a "correct answers" value
 
+### How to Score Individual and Team Points
+**Individual Points**
+
+_**Points Earned**_
+- +20 Points for every correct question & toss-up question
+- +10 Points for every quiz out without error
+
+**_Points Deducted_**
+- -10 Points for 2nd and subsequent personal errors
+     - Points are not deducted for errors on bonus questions
+     - Points are not deducted for errors during overtime
+- -10 Points for 3rd personal foul
+
+
+**Team Points**
+All point earned or deducted by an individual are to be counted towards the team’s points.
+
+_**Points Earned**_
+- +20 Points for team arriving on time. 
```

Comment:
I would phrase it differently. Perhaps vague as "being on time" or more specific "being present on time"

Arriving is too squishy IMO

Reply:
+20 Points for each team present at the scheduled start time of the quiz.